tools/runner/client.go

```diff
@@ -0,0 +1,70 @@
+package runner
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	// This side-effect import is required by GKE.
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	grpcv1 "github.com/grpc/test-infra/api/v1"
+	clientset "github.com/grpc/test-infra/clientset"
+)
+
+// NewLoadTestGetter returns a client to interact with LoadTest resources.
+// The client can be used to create, query for status and delete LoadTests.
+func NewLoadTestGetter() clientset.LoadTestGetter {
+	schemebuilder := runtime.NewSchemeBuilder(func(scheme *runtime.Scheme) error {
+		scheme.AddKnownTypes(grpcv1.GroupVersion,
+			&grpcv1.LoadTest{},
+			&grpcv1.LoadTestList{},
+		)
+		metav1.AddToGroupVersion(scheme, grpcv1.GroupVersion)
+		return nil
+	})
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		if err != rest.ErrNotInCluster {
+			log.Fatalf("failed to connect within cluster: %v", err)
```

Comment:
I am a bit confused, in this case do we regard the message in log.Fatalf as error message or information message?

Reply:
This code came from the example under cmd/clientset. We should improve it to not use log.Fatalf (it should probably return an error instead of exiting), but in this case it is only caused once from main(), so the effect is the same. I would like to defer looking at this until after we merge this PR.